src/main/java/games/strategy/engine/lobby/server/db/Database.java

```diff
@@ -123,7 +123,7 @@ private static void ensureDbTablesAreCreated(final Connection conn) {
         if (!existing.contains("TA_USERS")) {
           final Statement s = conn.createStatement();
           s.execute("create table ta_users" + "(" + "userName varchar(40) NOT NULL PRIMARY KEY, "
-              + "password varchar(40) NOT NULL, " + "email varchar(40) NOT NULL, " + "joined timestamp NOT NULL, "
+              + "password varchar(60) NOT NULL, " + "email varchar(40) NOT NULL, " + "joined timestamp NOT NULL, "
```

Comment:
I'm assuming this code path only gets run in the context of the Derby database (because the Postgres database should always have its `ta_users` table pre-created).  However, do we need to worry about data migration for the lobby's existing Derby database if this change gets deployed while Derby is still active?